### ISSUE TITLE
Accessibility – only use buttons as click targets

### ISSUE THREAD

**Issue by jrock2004** [balinterdi/rarwe-issues#439]
I am curious as maybe this is out of scope of the book, but some of the HTML you write is not following some of the practices of accessibility. An example of this is on page 70

```
<div class="rr-new-label">
  <label {{on "click" this.addBand}}>
    {{fa-icon "plus"}}
    <span class="ml1">Add new band</span>
  </label>
</div>
```

Maybe that is asking too much, if so that is ok as the book is not writing accessible code in ember. Just thought I would ask

**Comment by balinterdi** [comment]
No, you're right, low-hanging fruits for accessibility should absolutely be picked and that's such a case.

I'd converted a few cases of accessibility violation but probably missed a few. However, I didn't find the above code now. Could you give me the page number where it's found? (And are you reading the latest version, `20190913`?)

**Comment by jrock2004** [comment]
Yes its that version and page is actually 74

**Comment by balinterdi** [comment]
Thank you, I've fixed that one and found a few others that need to be fixed.
